core-galleon-pack/pom.xml

```diff
@@ -128,6 +128,11 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.spec.javax.security.auth.message</groupId>
+            <artifactId>jboss-jaspi-api_1.1_spec</artifactId>
```

Comment:
@darranl This also needs adding to the core-feature-pack/pom.xml

Reply:
Hi @kabir it is already there in the preparatory work https://github.com/wildfly/wildfly-core/blob/master/core-feature-pack/pom.xml#L123

Reply:
The core-galleon-pack one is also there, once I synchronise with upstream this should drop from this PR https://github.com/wildfly/wildfly-core/blob/master/core-galleon-pack/pom.xml#L115